EIPS/eip-6914.md

```diff
@@ -0,0 +1,77 @@
+---
+eip: 6914
+title: Reuse Withdrawn Validator Indices
+description: Reuse fully withdrawn and safe to reuse validator indices for new beacon chain deposits.
+author: Lion (@dapplion), Danny Ryan (@djrtwo)
+discussions-to: https://github.com/ethereum/consensus-specs/issues/3335
```

Comment:
I didn't realize that discussions-to *MUST* be eth magicians. Can we not use an issue on the spec repo? This is where organic conversation aroudn these specs already occurs.

Reply:
The main concern is about deletion of the issue, but i guess can be waived for `execution-specs` and `consensus-specs` issues as those two repos can enforce non deletion